builds/install/misc/firebird.conf

```diff
@@ -62,21 +64,20 @@
 # There is a number of predefined macro commands, that can be used in config
 # files where directory name is needed. They are available using $(name) syntax.
 # The complete list of them as follows:
-#	root - root directory of firebird instance
-#	install - directory where firebird is installed
+#	root - root directory of Firebird instance
+#	install - directory where Firebird is installed
 #	this - directory where current configuration file is located
-#	dir_conf - directory where firebird.conf and databases.conf are located
```

Comment:
Dislike this change - file is all-low letters.

Reply:
That is an error on my side, my intention wasn't to replace that, only occurrence of Firebird in isolation.

Reply:
Fixed in 11c7d480ac10006e804aa6a94ce1cac17f326f0b